api/idp/api.go

```diff
@@ -28,8 +28,8 @@ type IdentityProviderRepresentation struct {
 }
 
 type HrdSettingModel struct {
-	IPRangesList string `json:"ipRangesList"`
-	Priority     int    `json:"priority"`
+	IPRangesList *string `json:"ipRangesList"`
```

Comment:
I think we should have field domainsList to avoid losing its value if it is already set.